### DISCUSSION TITLE
BestDose assay error

### DISCUSSION THREAD

**Discussion by sumithmathew9** [LAPKB/Pmetrics#225] [Q&A] [answered]
I am aware that Pmetrics use assay error information to strip away process noise and thereby help with estimation of true joint probability distribution. However, TDM units using BestDose also have their own assay error. 
Does BestDose quantify the assay error of the laboratory, while using it for dose optimisation? 
Does BestDose use the assay error polynomial of the PK model used as Bayesian prior and give weightage for the concentrations estimated accordingly?

**Comment by mnneely** [comment, marked as answer]
Currently the web version of BestDose uses the assay error in the model not the individual lab. We recognize this isn’t theoretically ideal, although in practice we haven’t found it to significantly hamper performance as most laboratories have assays with similar performance, <10%CV and censoring, which is a whole other topic! :)
